### DISCUSSION TITLE
Resizable property in navbar

### DISCUSSION THREAD

**Discussion by GermanJablo** [mantinedev/mantine#2048] [Feature requests]
In [this related discussion ](https://github.com/mantinedev/mantine/discussions/1691
) it has been suggested to use third party libraries to handle resizing.

However, I think it would be nice if the Navbar component had a "resizable" boolean property, since it is a very frequent use case. 

It could be done without libraries, just with a very light js and css. Examples of implementations (the principle is essentially the same):

- https://codepen.io/mcolo/pen/OJMjWda?editors=1010
- https://stackoverflow.com/questions/38874252/how-to-drag-the-sidebar
- https://stackoverflow.com/questions/60981769/resizable-sidebar-drag-to-resize

Ideally, a maxWidth and minWidth could be defined.

**Comment by rtivital** [comment]
Hi @EgonBolton, thanks for a feature request. Unfortunately, this seems to be outside of current scope (we do not have time to implement/maintain it any time soon), maybe it will be added in one of the 6.x releases.

**Reply by GermanJablo** [reply]
Thanks for your prompt response. Well, if that's the case, I'll see if I can do it myself and send the PR.
